data/summary/store/store.go

```diff
@@ -202,11 +202,11 @@ func (r *Repo[T, A]) SetOutdated(ctx context.Context, userId, reason string) (*t
 	}
 
 	if userSummary == nil {
-		userSummary = types.Create[T, A](userId)
+		userSummary = types.Create[A](userId)
```

Comment:
There are not tests on this method, it might be time to add one or two.

Reply:
There are, but only the most basic, just confirming that it results in a summary of the right type, none of the other dependencies, ill add some deeper tests.

data/summary/types/summary_[cgm|bgm]_test.go has one test each.